classes/vm-image-features.bbclass

```diff
@@ -6,6 +6,7 @@ inherit core-image
 
 FEATURE_PACKAGES_qemu-virtio = "\
     kernel-module-sel4-virt \
+    kmod-sel4-tracebuffer \
```

Comment:
Could we use name prefix `kernel-module` here as well?

Reply:
Sure, NP